api/managementpb/json/managementpb.json

```diff
@@ -4490,7 +4490,7 @@
         "tags": [
           "SecurityChecks"
         ],
-        "summary": "ListSecurityChecks returns a list of available Security Thread Tool checks and their statuses.",
+        "summary": "ListSecurityChecks returns a list of available Security Thread Tool checks, their statuses and description.",
```

Comment:
This comment is outdated. Please regenerate code

Reply:
done